gslib/utils/copy_helper.py

```diff
@@ -3586,18 +3621,32 @@ def _CreateManifestFile(self):
       if ((not os.path.exists(self.manifest_path))
           or (os.stat(self.manifest_path).st_size == 0)):
         # Add headers to the new file.
-        with open(self.manifest_path, 'wb', 1) as f:
-          writer = csv.writer(f)
-          writer.writerow(['Source',
-                           'Destination',
-                           'Start',
-                           'End',
-                           'Md5',
-                           'UploadId',
-                           'Source Size',
-                           'Bytes Transferred',
-                           'Result',
-                           'Description'])
+        if six.PY3:
```

Comment:
Can we factor this list out (exactly as it is, making sure to keep the order and all)?  We type it twice and take up lots of lines for no reason.